### DISCUSSION TITLE
bearerAuth and/or ApiKeyAuth

### DISCUSSION THREAD

**Discussion by sebastianpelowski** [utmstack/UTMStack#839] [Q&A] [answered]
Is there a documented procedure to generate ApiKeyAuth or bearerAuth key to integrate UTMStack API with DFIR-IRIS or another IRS system?

I have been searching for this in https://github.com/utmstack/Documentation/ but I think there is a lack of information.

**Comment by c3s4rfred** [comment, marked as answer]
Hi @sebastianpelowski, UTMStack don't have specific integrations for IRS systems, but, if the IRS system can send logs via syslog, can be integrated and monitored using UTMStack platform. Please contact our sales team to know better, the capabilities of our platform.

Best regards
